backstop/tests/toggle-switch.html

```diff
@@ -150,142 +155,146 @@ <h2>Default</h2>
-          flex-direction: column;
-          align-items: flex-end;"
-      >
-        <label class="iui-toggle-switch">
-          <input
-            type="checkbox"
-            checked
-          />
-          <span class="iui-label">Option 1</span>
-          <span class="iui-toggle">
-            <span class="iui-handle"></span>
-          </span>
-        </label>
-        <label class="iui-toggle-switch">
-          <input type="checkbox" />
-          <span class="iui-label">Option 2</span>
-          <span class="iui-toggle">
-            <span class="iui-handle"></span>
-          </span>
-        </label>
-        <label class="iui-toggle-switch iui-disabled">
-          <input
-            type="checkbox"
-            checked
-            disabled
-          />
-          <span class="iui-label">Option 3</span>
-          <span class="iui-toggle">
-            <span class="iui-handle"></span>
-          </span>
-        </label>
-        <label class="iui-toggle-switch iui-disabled">
-          <input
-            type="checkbox"
-            disabled
-          />
-          <span class="iui-label">Option 4</span>
-          <span class="iui-toggle">
-            <span class="iui-handle"></span>
-          </span>
-        </label>
+      <div class="iui-toggle-switch-wrapper">
+        <input
+          class="iui-toggle-switch"
+          type="checkbox"
+          role="switch"
+          checked
+          disabled
+        />
+        <svg-placeholder
+          class="iui-toggle-switch-icon"
+          aria-hidden="true"
+        ></svg-placeholder>
       </div>
 
-      <h3>No label</h3>
-      <div id="no-label">
-        <label class="iui-toggle-switch">
-          <input
-            type="checkbox"
-            checked
-          />
-          <span class="iui-toggle">
-            <span class="iui-handle"></span>
-          </span>
-        </label>
-        <label class="iui-toggle-switch">
-          <input type="checkbox" />
-          <span class="iui-toggle">
-            <span class="iui-handle"></span>
-          </span>
-        </label>
-        <label class="iui-toggle-switch iui-disabled">
-          <input
-            type="checkbox"
-            checked
-            disabled
-          />
-          <span class="iui-toggle">
-            <span class="iui-handle"></span>
-          </span>
-        </label>
-        <label class="iui-toggle-switch iui-disabled">
-          <input
-            type="checkbox"
-            disabled
-          />
-          <span class="iui-toggle">
-            <span class="iui-handle"></span>
-          </span>
-        </label>
+      <div class="iui-toggle-switch-wrapper">
+        <input
+          class="iui-toggle-switch"
+          type="checkbox"
+          role="switch"
+          disabled
+        />
+        <svg-placeholder
+          class="iui-toggle-switch-icon"
+          aria-hidden="true"
+        ></svg-placeholder>
       </div>
+      </label>
```

Comment:
This might be leftover

Reply:
You are correct, removed from the next push.